apps/wallet/src/ui/app/pages/home/apps/index.tsx

```diff
@@ -1,33 +1,65 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import { useFeature } from '@growthbook/growthbook-react';
 import { Route, Routes } from 'react-router-dom';
 
 import { Content } from '_app/shared/bottom-menu-layout';
 import FiltersPortal from '_components/filters-tags';
 import AppsPlayGround, { ConnectedAppsCard } from '_components/sui-apps';
+import { FEATURES } from '_src/shared/experimentation/features';
+
+import type { DAppEntry } from '_src/ui/app/components/sui-apps/SuiApp';
 
 import st from './AppsPage.module.scss';
 
+type FilterTag = {
+	name: string;
+	link: string;
+};
+
 function AppsPage() {
-	const filterTags = [
+	const defaultFilterTags: FilterTag[] = [
 		{
-			name: 'Playground',
-			link: 'apps',
+			name: 'Connections',
+			link: 'apps/connected',
 		},
 		{
-			name: 'Active Connections',
-			link: 'apps/connected',
+			name: 'All',
+			link: 'apps',
 		},
 	];
+	const ecosystemApps = useFeature<DAppEntry[]>(FEATURES.WALLET_DAPPS).value ?? [];
+
+	const uniqueAppTagNames = new Set<string>();
+
+	ecosystemApps
+		.flatMap((app) => app.tags)
+		.filter((tag) => {
+			if (uniqueAppTagNames.has(tag)) {
+				return false;
+			}
+
+			uniqueAppTagNames.add(tag);
+			return true;
+		});
+
+	const uniqueAppTags = [...uniqueAppTagNames].map((tag) => ({
```

Comment:
```suggestion
	const uniqueAppTags = Array.from(new Set(ecosystemApps.flatMap((app) => app.tags))).map((tag) => ({
```
I think something like the above is a bit more readable